### ISSUE TITLE
Fuzzer - Export results

### ISSUE THREAD

**Issue by zapbot** [zaproxy/zaproxy#1153]
```
Hello, is there a way to export (ideally to csv format) the results from the fuzzer
that are on the fuzzer tab? I don't need the responses or the requests, just the list
with the results. 


Thanks,
Yiannis 

```

Original issue reported on code.google.com by `yianniskapsalis` on 2014-04-14 20:39:38

**Comment by mustafa421** [comment]
I'd love to try and work on this as a 'firstbug' to familiarize myself with the code as a student. Could anyone possible point me in the right direction to get started? @psiinon

**Comment by thc202** [comment]
The first step is to fork and clone the zap-extension repo. [1]
That repo contains all the ZAP add-ons (maintained by ZAP dev team).
The zap-extensions repo has three branches master, beta and alpha for add-ons in release, beta and alpha status, respectively.

The fuzzer add-on is currently in beta status so you have to checkout the beta branch.
The classes for that add-on are under the fuzz package. [2]

The interesting classes to fix this issue are under the package httpfuzzer, which has the HTTP fuzzer implementation. [3]

I'd expect this option to be added to the results panel. [4]
From that panel/class you can access the currently selected fuzzer to obtain and export the results.

Let us know if you have any other question.

[1] https://github.com/zaproxy/zap-extensions
[2] https://github.com/zaproxy/zap-extensions/tree/1a4b58bde485b810cf4e02ada218e887f8de7581/src/org/zaproxy/zap/extension/fuzz
[3] https://github.com/zaproxy/zap-extensions/tree/1a4b58bde485b810cf4e02ada218e887f8de7581/src/org/zaproxy/zap/extension/fuzz/httpfuzzer
[4] https://github.com/zaproxy/zap-extensions/blob/1a4b58bde485b810cf4e02ada218e887f8de7581/src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzResultsContentPanel.java

**Comment by mustafa421** [comment]
Review my pull request [here](https://github.com/zaproxy/zap-extensions/pull/284) for this issue

**Comment by mustafa421** [comment]
@kingthorin wouldn't this have to be closed as well?

**Comment by thc202** [comment]
Fixed in zaproxy/zap-extensions#284.

**Comment by thc202** [comment]
Released in version 6 of Fuzzer add-on.
